app/javascript/components/NamedAvatar.js

```diff
@@ -33,14 +33,13 @@ const Loader = _props => (
   </Container>
 )
 
-const NamedAvatar = ({ loading, image, name, subtitle }) => {
+const NamedAvatar = ({ loading, image, name }) => {
   if (loading) return <Loader />
   return (
     <Container>
       <Avatar image={image} />
       <Details>
         <MD tag="strong">{name}</MD>
-        <SM>{subtitle || '\u00a0'}</SM>
```

Comment:
instead of removing the functionality entirely, can we just not pass it in?

Reply:
Yep, that'll be easier to hook up when we do get the departments to sync, I just went on a cleaning spree 😄 i'll change this